extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -96,6 +96,8 @@ export class AmpAdXOriginIframeHandler {
           this.element_.creativeId = info.data.id;
         });
 
+    // TODO(keithwrightbos) - move to within not A4A else block to ensure this
```

Comment:
TODO is because getHTML is arguably only for ad tags.  For Fast fetch, no ad tag executes so no need to expose